### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by AbrarNitk**
[Change Request](https://www.fifthtry.com/fifthtry/ft-sync/~/6/)

### CODE REVIEW

File: docs/index.ftd [hunk 1]
```diff
@@ -0,0 +1,64 @@
+-- meta:
+git-repo: a/b
```
Comment (amitu) [inline docs/index.ftd:2]: This is wrong: must point to this repo.

**Comment by amitu** [comment]
You ll have to update the test files too.

File: docs/index.ftd (hunk 1 above)
Comment (AbrarNitk) [reply docs/index.ftd:2]: This is just for local test purpose

**Comment by amitu** [comment]
Attach to a CR please.

**Comment by AbrarNitk** [comment]
Testing `status` first, If it's works fine, `sync` will also work

File: .github/workflows/ft-sync.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  everything:
+    name: ft-sync
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Running ft-sync status
+#      - uses: actions/checkout@v1
+        run: |
+          ls -la
+          chmod +x ./ft-sync
```
Comment (amitu) [inline .github/workflows/ft-sync.yml:18]: Move the rust binary to .github folder. Also append platform name to binary, `ft-sync-linux-x64` or something.

**Comment by amitu** [comment]
And this entire action is not for this repo, its for your ft-sync-demo repo.

File: .github/workflows/ft-sync.yml [hunk 3]
```diff
@@ -9,6 +9,8 @@ jobs:
   everything:
     name: ft-sync
     runs-on: macos-10.15
+    env:
+      FT_AUTH_CODE: abcd
```
Comment (amitu) [inline .github/workflows/ft-sync.yml:13]: Nopes, to send ENV you have to update github settings. You cant store the auth code in publicly visible place.

File: .github/workflows/ft-sync.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+on:
```
Comment (amitu) [inline .github/workflows/ft-sync.yml:1]: Remove this entire file and put it in your sync demo repo.

File: docs/hello-world.ftd [hunk 5]
```diff
@@ -9,4 +9,8 @@ Hi My name is Abrar Khan
 Hi My name is Abrar Khan
 
 
-Hello world testing
\ No newline at end of file
+Hello world testing
+
+
+
+
```
Comment (amitu) [inline docs/hello-world.ftd:16]: Delete this entire folder. Whatever you need for testing move to a folder inside tests folder.

File: ft-sync.p1 [hunk 6]
```diff
@@ -2,8 +2,8 @@
 mode: local-to-remote
 backend: ftd
 root: docs
-repo: amitu/fifthtry
-collection: testuser/index
+repo: https://github.com/AbrarNitk/ft-sync-demo
+collection: wilderbit/ft-sync-demo
```
Comment (amitu) [inline ft-sync.p1:6]: Delete this file.

File: .github/workflows/ft-sync.yml (hunk 4 above)
Comment (AbrarNitk) [reply .github/workflows/ft-sync.yml:1]: Yeah doing that only

**Review by amitu — APPROVED** [review]

**Pull request merged**